src/components/views/rooms/MemberInfo.js

```diff
@@ -481,22 +489,22 @@ module.exports = WithMatrixClient(React.createClass({
     },
 
     _calculateCanPermissions: function(me, them, powerLevels) {
```

Comment:
/me wishes this or something more generic for figuring out the client's abilities was pushed way further down the stack, potentially to the js-sdk